pages/agent/v3/plugins.md.erb

```diff
@@ -73,27 +73,33 @@ changing unexpectedly. This is done with a git tag of `vX.X.X`.
 
 The best place to get started is the [Buildkite Plugins] repository. You can also find third-party plugins with the [buildkite-plugin tag on GitHub][third party plugins].
 
-At Buildkite, our plugins are predominantly shell scripts, so we've created a [plugin tester] docker image that simplifies writing BATS tests and getting your plugin tested (so meta).
-
 ### Anatomy of a plugin
 
 Plugins are essentially a collection of hooks with extra configuration. The agent parses the plugin configuration and passes it on to the plugin in a slightly normalized form, both as JSON or environment variables.
 
 This means that you can write a plugin as a shell script, or as a binary.
 
-A minimal plugin has a `plugin.json` at it's root:
+A plugin has a `plugin.yml` at it's root that describes what it does, the executables required on the agent machine, and a schema of the plugin’s configuration options:
```

Comment:
Should we mention that this used to be a plugin.json? This could go in a troubleshooting note at the end of the section?

Reply:
Hrmmm… good questions. I don't think so. The plugin community is still small, and and I've sent pull requests to many of them already. But actually there's still a few we should help to update… I'll go do that now.